packages/enzyme/src/ReactWrapper.js

```diff
@@ -446,7 +454,10 @@ class ReactWrapper {
    * @returns {Boolean}
    */
   containsAnyMatchingElements(nodes) {
-    return Array.isArray(nodes) && nodes.some(node => this.containsMatchingElement(node));
+    const rstNodes = nodes.map(getAdapter().elementToNode);
```

Comment:
this change makes `.map` be called before `Array.isArray` is called to ensure it's an array.

Let's change this to `if (!Array.isArray(nodes)) { return false; }` at the top of the function, and then we can do the entire check in a single `some`